src/overlays/actors/ovl_Bg_Last_Bwall/z_bg_last_bwall.h

```diff
@@ -2,16 +2,35 @@
 #define Z_BG_LAST_BWALL_H
 
 #include "global.h"
+#include "objects/object_last_obj/object_last_obj.h"
 
 struct BgLastBwall;
 
 typedef void (*BgLastBwallActionFunc)(struct BgLastBwall*, PlayState*);
 
+#define BGLASTBWALL_GET_SWITCHFLAGS(thisx) (((thisx)->params & 0x7F))
+#define BGLASTBWALL_GET_F000(thisx) (((thisx)->params >> 0xC) & 0xF)
+
+typedef struct {
+    Vec3s* unk0;
+    s16* unk4;
+    s32 unk6;
+} InitColliderStruct;
+
+typedef struct {
+    Gfx* dList;
+    s32 unk4;
+    CollisionHeader* colHeader;
+} ModelInfo;
```

Comment:
A few notes about both these structs:
1. Could you move them to the `.c` file since really only this actor will use them?
2. Could you append the Actor name to the name of each struct to avoid conflicts? e.g `ModelInfo` -> `BgLastBwallModelInfo`
3. Could you add offset comments to each struct member on the left side? e.g. `/* 0x0 */ Gfx* dList;`
4. Last could you add a size comment? e.g. `ModelInfo; // size = 0x10`